content/de/contribute/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Wie Man etwas beitragen kann
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-pen-square'></i>
+---
+
+Der Inhalt des Cloud Native Glossars ist [hier](https://github.com/cncf/glossary) gespeichert, wo sich eine Liste von [GitHub Issues](https://github.com/cncf/glossary/issues), [Pull Requests](https://github.com/cncf/glossary/pulls) und [Diskussionen](https://github.com/cncf/glossary/discussions) über das Glossar finden.
+
+Es gibt drei Möglichkeiten sich zu beteiligen:
+
+1) Neue Begriffe vorschlagen
+2) Existierende Begriffe aktualisieren 
+3) Bei der Übersetzung helfen
+
+## Werde Mitglied der Glossar-Community!
+Wer regelmäßig einen Beitrag leisten möchten, sollte an den monatlichen Treffen der Glossar-Arbeitsgruppe teilnehmen. Details zu den Treffen befinden sich im CNCF-Kalender. Ihr könnt euch auch mit den Betreuern und anderen Mitwirkenden in unserem #glossary Kanal über den [CNCF Kalender] (https://www.cncf.io/calendar/) austauschen - wir freuen uns auf euch! 
```

Comment:
Oops, my bad. We should change to*:

Wer regelmäßig einen Beitrag leisten möchten, sollte an den monatlichen Treffen der Glossar-Arbeitsgruppe teilnehmen. Details zu den Treffen befinden sich I'm [CNCF-Kalender](https://www.cncf.io/calendar/). Ihr könnt euch auch mit den Betreuern und anderen Mitwirkenden in unserem #glossary Kanal über den Slack austauschen - wir freuen uns auf euch!